api/models/workflow.py

```diff
@@ -108,7 +108,7 @@ class Workflow(db.Model):
     )
     updated_by: Mapped[Optional[str]] = mapped_column(StringUUID)
     updated_at: Mapped[datetime] = mapped_column(
-        sa.DateTime, nullable=False, default=datetime.now(tz=UTC), server_onupdate=func.current_timestamp()
+        sa.DateTime, nullable=False, server_onupdate=db.text("CURRENT_TIMESTAMP(0)")
```

Comment:
Can you explain why this modification was made? What is the difference between using db.text and func.current_timestamp?

Reply:
The biggest difference is the removal of the default value to avoid updated_at being the server startup time. 
I think func.current_timestamp() is better than db.text(‘CURRENT_TIMESTAMP(0)’) for cross-database auto-adaptation. 
So I replaced both created_at and updated_at

Reply:
updated_at use server_default instead of default

Reply:
![image](https://github.com/user-attachments/assets/56a91a67-105e-4c12-91f6-f678c1500137)